tests/application-test.js

```diff
@@ -754,6 +754,18 @@ describe('Box.Application', function() {
 				Box.Application.broadcast('abc', messageData);
 			});
 
+			it('should call handler of onmessage object', function() {
```

Comment:
by convention, we use "should ____ when" so this should be something like "should call corresponding message handler when a message is received'